app/graphql/types/media_reaction_vote.rb

```diff
@@ -0,0 +1,14 @@
+class Types::MediaReactionVote < Types::BaseObject
```

Comment:
Should this match the approach on Likes, where you just have a list of users?

Reply:
yeah it should, I'll fix it up.